selfdrive/car/hyundai/hyundaicanfd.py

```diff
@@ -61,9 +61,12 @@ def create_steering_messages(packer, CP, CAN, enabled, lat_active, apply_steer):
   return ret
 
 def create_cam_0x2a4(packer, CAN, cam_0x2a4):
-  values = {f"BYTE{i}": cam_0x2a4[f"BYTE{i}"] for i in range(3, 24)}
-  values['COUNTER'] = cam_0x2a4['COUNTER']
-  values["BYTE7"] = 0
+  values = {f"BYTE{i}": cam_0x2a4[f"BYTE{i}"] for i in range(3, 24) if i != 7}
+  values["COUNTER"] = cam_0x2a4["COUNTER"]
+  values["NEW_SIGNAL_1"] = cam_0x2a4["NEW_SIGNAL_1"]
+  values["NEW_SIGNAL_2"] = cam_0x2a4["NEW_SIGNAL_2"]
```

Comment:
Any vague ideas for the new signals?

Reply:
They're fillers in between the 2 laneline signals, they stay at 0 in all the routes I've checked. Ioniq 6 HDA2 and Kona EV 2nd Gen HDA2 tested with this being copied and forwarded and LFA still stays suppressed. Do we want to remove them if they're insignificant to copy and forward?